app/lib/common/widget/NewsSideBar.dart

```diff
@@ -1,5 +1,6 @@
 // ignore_for_file: prefer_const_constructors
 
+import 'package:cached_network_image/cached_network_image.dart';
```

Comment:
I don't understand what these changes have to do with the chat-PR. Please revert this. If they are relevant please raise them as a separate PR.

Reply:
This relates to throwing bug if we don't use cached image in news feed avatar. Although its unrelated, I'll remove that.

Reply:
@gtalha07 leave it in, just a comment for later to keep things one-thing-per-PR.